modules/client-configuration/pages/package-management.adoc

```diff
@@ -47,3 +47,51 @@ When the profile has been created, you can use it to install more clients with t
 . In the {productname} {webui}, navigate to [guimenu]``Systems``, click the client to compare, and navigate to the menu:Systems[Software > Packages > Profiles] subtab.
     To compare with a stored profile, select the profile and click btn:[Compare].
 . To compare with another client, select the client name and click btn:[Compare] to see a list of differences between the two clients.
+
+
+
+== Orphaned Packages
+
+Orphaned packages do not have associated channels.
```

Comment:
WDYT about defining the term? You write that orphaned packages do not have associated channels, but maybe someone could be confused if "orphaned package" is a new term, or if it's a type of "package"? 

```suggestion
Orphaned packages are packages synchronized by {productname} that are not associated with a software channel.
```